generator/config/expression/isArray.yaml

```diff
@@ -3,12 +3,31 @@ name: $isArray
 link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/isArray/'
 type:
     - resolvesToBool
-encode: array
+encode: single
```

Comment:
The syntax [in the doc](https://www.mongodb.com/docs/manual/reference/operator/aggregation/isArray/) is confusing. It looks like the expression must be encapsulated in a single-value array, but in the example, the variable is used directly.
Maybe a doc issue.

Reply:
I believe this is covered by this note in the docs:
> Aggregation expressions accept a variable number of arguments. These arguments are normally passed as an array. However, when the argument is a single value, you can simplify your code by passing the argument directly without wrapping it in an array.

I'd have to do some tests on how values are handled. Looking at the examples in the [Behavior](https://www.mongodb.com/docs/manual/reference/operator/aggregation/isArray/#behavior) section, we should make sure that passing a value of `[1]` is actually recognised as an array, which might require changing `encode` back to `array` and updating the examples in the config files.

Reply:
Ok, reverted to always use the array syntax.